genie-web/src/main/java/com/netflix/genie/web/services/JobDirectoryServerService.java

```diff
@@ -17,12 +17,11 @@
  */
 package com.netflix.genie.web.services;
 
+import com.netflix.genie.common.exceptions.GenieException;
```

Comment:
I'd like to get away from using the exceptions in the old common module as we move towards hopefully deleting it at the completion of v4 development. It doesn't have to be part of this commit but we should move to exceptions in either `common-external`, `common-internal`, `web` or `agent` so they're scoped properly.